CMakeLists.txt

```diff
@@ -47,11 +47,13 @@ endif()
 # specify the C++ standard
 set(CMAKE_CXX_STANDARD 14)
 
+add_subdirectory(third_party)
 add_subdirectory(src)
 
+add_subdirectory(unit_tests)
 if(WIN32 AND USE_WIN32_BMP)
-  message(WARNING "USE_WIN32_BMP is enabled.  Therefore testing is disabled.")
-  message(WARNING "To enable testing invoke with `cmake -DUSE_WIN32_BMP=NO`")
+  message(WARNING "USE_WIN32_BMP is enabled.  Therefore regression testing is disabled.")
+  message(WARNING "To enable regression testing invoke with `cmake -DUSE_WIN32_BMP=NO`")
```

Comment:
Oh - that's a bug.  In fact `USE_WIN32_BMP` is no longer relevant, as we have the `/BM:fmt` command line option.

So don't worry about that change - I'll do something to drop it.

Reply:
Reverted